### PR TITLE
use androidAdbPort cap to specify a custom adb port

### PR DESCRIPTION
**Pull request by jlipps**

### CODE REVIEW

**Comment by imurchie** [comment]
Is the `android` in `androidAdbPort` not redundant?

**Comment by jlipps** [comment]
updated!

File: lib/driver.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import { fs, tempDir } from 'appium-support';
 
 const APP_EXTENSION = '.apk';
 const DEVICE_PORT = 4724;
+const DEFAULT_ADB_PORT = 5037;
```
Comment (imurchie) [inline lib/driver.js]: Should this be exposed from `appium-adb` and just used here? Seems like info only it should know.

**Comment by jlipps** [comment]
ran the failing functional locally and it passed, going ahead and merging

**Comment by jlipps** [comment]
published as 1.6.12

**Comment by jlipps** [comment]
i mean, 1.6.11 not counting the fat finger

**Pull request merged**